docs/user/openstack/observability.md

```diff
@@ -0,0 +1,381 @@
+      - url: "https://external-prometheus.example/api/v1/write"
+        writeRelabelConfigs:
+        - sourceLabels:
+          - __tmp_openshift_cluster_id__
+          targetLabel: cluster_id
+          action: replace
+        tlsConfig:
+          ca:
+            secret:
+              name: mtls-bundle
+              key: ca.crt
+          cert:
+            secret:
+              name: mtls-bundle
+              key: ocp-client.crt
+          keySecret:
+            name: mtls-bundle
+            key: ocp-client.key
+```
+
+Save it to a file named `cluster-monitoring-config.yaml`. Before applying it,
+create the secret containing the HTTPS client certificates, similar to what we
+did for RHOSO. We're still calling the secret `mtls-bundle`, but this time in
+the `openshift-monitoring` namespace:
+
+```bash
+oc --namespace openshift-monitoring \
+    create secret generic mtls-bundle \
+        --from-file=./ca.crt \
+        --from-file=ocp-client.crt \
+        --from-file=ocp-client.key
+```
+
+Once you have created the secret, it's time to apply the cluster-monitoring configuration:
+
+```bash
+oc apply -f cluster-monitoring-config.yaml
+```
+
+Let the change propagate and verify that you receive OpenShift metrics in the
+external Prometheus.
+
+[ocp_docs]: https://docs.openshift.com/container-platform/4.17/observability/monitoring/configuring-the-monitoring-stack.html#configuring_remote_write_storage_configuring-the-monitoring-stack "Configuring the monitoring stack"
+[create_cluster_monitoring_config]: https://docs.openshift.com/container-platform/4.17/observability/monitoring/configuring-the-monitoring-stack.html#creating-cluster-monitoring-configmap_configuring-the-monitoring-stack "Creating a cluster monitoring config map"
+[setup_remote_write]: https://docs.openshift.com/container-platform/4.17/observability/monitoring/configuring-the-monitoring-stack.html#configuring-remote-write-storage_configuring-the-monitoring-stack "Configuring remote write storage"
+[add_labels]: https://docs.openshift.com/container-platform/4.17/observability/monitoring/configuring-the-monitoring-stack.html#adding-cluster-id-labels-to-metrics_configuring-the-monitoring-stack "Adding cluster ID labels to metrics"
+
+### Method B: Scrap OCP metrics from RHOSO
+
+As an alternative to write the different metrics to a single store, it's
+possible to concentrate the storage of metrics to RHOSO's existing
+telemetry-operator.
+
+OpenShift exposes a federation endpoint to expose a subset of metrics to an
+external scraper. You can follow [these instructions][federation] to get
+acquainted to the endpoint.
+
+[federation]: https://docs.redhat.com/en/documentation/openshift_container_platform/4.16/html/monitoring/accessing-third-party-monitoring-apis#monitoring-querying-metrics-by-using-the-federation-endpoint-for-prometheus_accessing-monitoring-apis-by-using-the-cli "OpenShift documentation: Querying metrics by using the federation endpoint for Prometheus"
```

Comment:
Can you change the link to point to the latest OCP release?

Reply:
I'll give you 4.17 last offer